docs/KERBEROS.rst

```diff
@@ -13,49 +13,44 @@ Kerberos authentication
 
 Introduction
 ============
-Fledge implements through his North plugin PI_server Token, Basic and Kerberos authentication, the latter is especially relevant for the integration with PI Web API using `OMF`_.
+
+The bundled OMF north plugin in Fledge can use a number of different authentication schemes when communicating with the various OSIsoft products. The PI Web API method in the `OMF`_ plugin supports the use of a Kerberos scheme.
```

Comment:
the omf link will break  `.. _OMF: https://omf-docs.readthedocs.io/en/v1.1/` looks it is https://omf-docs.osisoft.com/#omf-specification-11
better is just to keep https://omf-docs.osisoft.com/